# Definitions of traceable bebop quadrotor
# Identifier corresponds to Traceable ID set in MoCap

node_rate: 100
delta_t: 0.05
horizon_N: 30
bebop2:
      pose_measured: /Bebop1/pose                                 # subscriber topic name
      full_state_estimation: /Bebop1/bebopFullStateEstimation     # publisher topic name